### DISCUSSION TITLE
Disable typecheck

### DISCUSSION THREAD

**Discussion by jacobdonenfeld** [golangci/golangci-lint#2140] [Q&A] [answered]
Is there any way to disable typecheck. It is detecting many issues that are not actual issues, ie:
imported but not used
undefined (type *var has no field or method T) 
declared but not used 
undeclared name
is unused
could not import C

Any ways to fix this is appreciated!

**Comment by ldez** [comment, marked as answer]
Hello,

typecheck cannot be disabled because it's not a real linter, it's just a way to display "compilation" errors.

The main root cause of your kind of errors is a misuse of go modules.
